projects/e2e-tests/codecov_tests/src/lib.rs

```diff
@@ -353,114 +350,97 @@ pub extern "C" fn finish() -> i32 {
     );
 
     // Slice
-    check_result(
-        unsafe { host_bindings_loose::get_tx_nested_array_len(-1, locator.len() as i32) },
-        error_codes::INVALID_PARAMS,
-        "get_tx_nested_array_len_neg_ptr",
-    );
-    check_result(
-        unsafe { host_bindings_loose::get_tx_nested_array_len(locator.as_ptr() as i32, -1) },
-        error_codes::INVALID_PARAMS,
-        "get_tx_nested_array_len_neg_len",
-    );
     let long_len = 4 * 1024 + 1;
     check_result(
-        unsafe {
-            host_bindings_loose::get_tx_nested_array_len(locator.as_ptr() as i32, long_len as i32)
-        },
-        error_codes::DATA_FIELD_TOO_LARGE,
+        unsafe { host::get_tx_nested_array_len(locator.as_ptr(), long_len) },
+        error_codes::LOCATOR_MALFORMED,
         "get_tx_nested_array_len_too_long",
     );
-    check_result(
-        unsafe {
-            host_bindings_loose::get_tx_nested_array_len(
-                locator.as_ptr() as i32 + 1_000_000_000,
-                locator.len() as i32,
-            )
-        },
-        error_codes::POINTER_OUT_OF_BOUNDS,
-        "get_tx_nested_array_len_ptr_oob",
-    );
+    // NOTE: These tests are commented out because the WASM runtime catches the out of bounds
```

Comment:
LMK and I'll simply remove these commented out tests. Wanted to leave them in for now to make sure we evaluate this change-set properly.